vocabularyCollection/phase.md

```diff
@@ -1,10 +1,10 @@
 | id | phase | description | status |
 |:----- |:----- |:------ |:------ |
-| 0 | unknown | when the platform phases describe below do not fit the platfrom behaviour, or when the behaviour of the platform is unknown | pending |
-| 1 | ascent | the platform is moving up toward surface with no internal command on the pitch angle | pending |
-| 2 | descent | the platform is moving down toward targeted depth of operation with no internal command on the pitch angle | pending |
-| 3 | surfacing | the platform is drifting at the surface for communication, recovery or safety purpose | pending |
-| 4 | parking | the platform is maintained at a parking depth | pending |
-| 5 | inflexion | the platform is changing pitch angle to move from a phase to another | pending |
-| 6 | propelled | the platform momentum is increased by the propeller | pending |
-| 7 | transition | the platform is changing from one phase to another | pending |
+| 0 | unknown | when the platform phases describe below do not fit the platform behaviour, or when the behaviour of the platform is unknown | pending |
+| 1 | ascent | the platform is moving up - climbing | pending |
+| 2 | descent | the platform is moving down - diving | pending |
+| 3 | surfacing | the platform is at the surface for communication, recovery or safety purpose | pending |
+| 4 | parking | the platform is maintained at a parking or hover depth | pending |
+| 5 | inflexion | the platform is changing from dive to climb or climb to dive | pending |
+| 6 | propelled | the platform momentum is increased by the propeller(s) or thruster(s) | pending |
```

Comment:
"momentum is increased" would be incorrect if the glider is e.g. moving at a steady speed. I would use the broader, more accurate description "the platform is underway using propeller(s)/thruster(s)"

Reply:
changed text